### PR TITLE
[FCOS] OKD: inject a setting to use community operators only

### PR DESCRIPTION
**Pull request by vrutkovs**
OKD installs may not have full pull secret, so it should be setup to use community operators only

See https://github.com/orgs/openshift/projects/1#card-29162680

### CODE REVIEW

File: pkg/asset/manifests/openshift.go [hunk 1]
```diff
@@ -138,13 +139,16 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
...
+	communityOperatorsSetting := &openshift.CommunityOperators{}
 	dependencies.Get(
 		cloudCredsSecret,
 		kubeadminPasswordSecret,
+		communityOperatorsSetting,
 		roleCloudCredsSecretReader)
 
 	assetData := map[string][]byte{
 		"99_kubeadmin-password-secret.yaml": applyTemplateData(kubeadminPasswordSecret.Files()[0].Data, templateData),
+		"99_comminity_operators.yaml":       applyTemplateData(communityOperatorsSetting.Files()[0].Data, templateData),
```
Comment (LorbusChris) [inline pkg/asset/manifests/openshift.go]: _community_

Comment (vrutkovs) [reply pkg/asset/manifests/openshift.go]: :+1: fixed

**Comment by LorbusChris** [comment]
@vrutkovs I just noticed the spelling is wrong in the PR title as well :)
other than that
/approve

File: pkg/asset/templates/content/openshift/community-operators.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/templates/content"
+)
+
+const (
+	communityOperatorsFilename = "cluster-networkconfig-crd.yaml"
```
Comment (sgreene570) [inline pkg/asset/templates/content/openshift/community-operators.go]: is this the right filename? Wouldn't the community operators file be `community-operators.yaml `?

Comment (vrutkovs) [reply pkg/asset/templates/content/openshift/community-operators.go]: Good catch, fixed

**Comment by LorbusChris** [comment]
now the `comminity` snuck back into the commit message :D

**Comment by patrickdillon** [comment]
> OKD installs may not have full pull secret, so it should be setup to use community operators only

I'm not up-to-speed on the context for this PR, but it looks like it enables community operators for all installs--not just OKD--is that correct?

**Comment by vrutkovs** [comment]
>it looks like it enables community operators for all installs--not just OKD--is that correct?

For any install using `fcos` branch (note the target of the PR), yes. When it would be merged into master we'll come up with some kind of switch to disable FCOS/OKD-specific features

**Comment by vrutkovs** [comment]
/override ci/prow/e2e-vsphere
vSphere support is not yet fixed

**Comment by vrutkovs** [comment]
/override ci/prow/e2e-aws

Known issues and flakes

**Pull request merged**